test/AutoRest.TestServerLowLevel.Tests/body-array.cs

```diff
@@ -63,6 +65,16 @@ public class BodyArray : TestServerLowLevelTestBase
             CollectionAssert.AreEqual(new[] { new object[] { "1", "2", "3" }, new object[] { "4", "5", "6" }, new object[] { "7", "8", "9" } }, data);
         });
 
+        [Ignore("Example")]
```

Comment:
So looks like these tests are for compilation purpose?

Reply:
That's correct. Right now due to rigid limitation of test server, we cannot arbitrarily add test cases. I've run those test cases on local, and they can pass. But there are still errors due to not match.